README.markdown

```diff
@@ -1,5 +1,31 @@
 # What is GitX?
 
+GitX is an OS X (MacOS) Native graphical client for the `git` version control system.
+
+
+GitX has a long history of various branches and versions maintained by various people over the years.  This github org & repo are an attempt to consolidate and move forward with a current, common, community-maintained version.   See discussion at [**Forking, and plans for the future of GitX**](https://github.com/gitx/gitx.github.io/issues/1).
```

Comment:
I don't think we should hard code a link to the issue into the README, I think we should distill the results of the discussion, or link to more information in a file in the project.

Minor typgraphical issues (which match the old README):
* Hard wrap to 72 chars or so (Markdown will still show it as one paragraph, but it's easier to read the source)
* Don't put two spaces after a `.`